src/templates/Events.js

```diff
@@ -111,6 +111,11 @@ export default props => {
         !isEmpty(edges) &&
         values(
           mapObjIndexed((group, key) => {
+            /* map over group, update slug to not include the events path since we are already on the events page */
+            group = group.map(g => {
```

Comment:
This map quickly fixes the issue and I think it's good to ship asap.

I am wondering if we can replace `g.node.fields.slug` with absolute path in `gatsby-node/createPages.js` so that we do not need this processing in different places. As an example, we have a similar issue is [contributors page](https://amplify.aws/community/contributors) which can be fixed with the same approach.

Reply:
I just made PR https://github.com/aws-amplify/community/pull/315 that describes what I commented. Hope it helps.

Reply:
Thanks, your solution works great! I'm reverting this portion of my PR and will merge yours shortly.